pkg/resmgr/nri.go

```diff
@@ -112,37 +112,6 @@ func (p *nriPlugin) Configure(cfg, runtime, version string) (stub.EventMask, err
 	), nil
 }
 
-func (p *nriPlugin) syncWithNRI(pods []*api.PodSandbox, containers []*api.Container) ([]cache.Container, []cache.Container, error) {
-	m := p.resmgr
-
-	m.Info("synchronizing cache state with NRI/CRI runtime...")
-
-	add, del := []cache.Container{}, []cache.Container{}
-
-	_, _, deleted := m.cache.RefreshPods(pods)
```

Comment:
We can't omit calling this as this is the one which makes that new pods are properly injected into the cache and stale ones are removed.